webapp/src/features/intersections/map/utilities/message-utils.ts

```diff
@@ -178,7 +178,7 @@ export const parseBsmToGeojson = (bsmData: OdeBsmData[]): BsmFeatureCollection =
         type: 'Feature',
         properties: {
           ...bsm.payload.data.coreData,
-          odeReceivedAt: new Date(bsm.metadata.odeReceivedAt as string).getTime() / 1000,
+          odeReceivedAt: new Date(bsm.metadata.odeReceivedAt as unknown as string).getTime() / 1000,
```

Comment:
Was the first cast as string not working on its own?

Reply:
I changed the type of odeReceivedAt to a number, which is the converted type, but it comes in from the api as a string and then gets converted to a number. Technically there should be two type definitions, but since this is the only difference they stay the same.